classes.rb

```diff
@@ -0,0 +1,47 @@
+class Person
+  def initialize(id, name, age, parent_permission)
+    @id = id
+    @name = name
```

Comment:
- [ ] I suggest you generate random IDs in the constructor method of this Person Class. Use the code snippet below 👇🏾 

```ruby
@id = Random.rand(1..1000) # this will generate random id between 1 and 1000.
```

With this, you do not have to pass `id` as an argument into the constructor method.

Reply:
well resolved! ✔️